tslib.d.ts

```diff
@@ -23,8 +23,11 @@ export declare function __generator(thisArg: any, body: Function): any;
 export declare function __exportStar(m: any, o: any): void;
 export declare function __values(o: any): any;
 export declare function __read(o: any, n?: number): any[];
+/** @deprecated since TypeScript 4.2 */
 export declare function __spread(...args: any[][]): any[];
+/** @deprecated since TypeScript 4.2 */
 export declare function __spreadArrays(...args: any[][]): any[];
+export declare function __spreadArray(to: any[], from: any[]): any[];

```

Comment:
This is more correct:
```suggestion
export declare function __spreadArray<T, U>(to: T[], from: ReadonlyArray<U>): Array<T | U>;
```

Although this would be even more correct, it depends on **TypeScript 4.0**:
```ts
export declare function __spreadArray<T extends any[], U extends readonly any[]>(to: T, from: U): [...T, ...U];
```

Reply:
The types here are fairly simplistic. If we were going to add generics here we would want to add them to other functions as well, but that's out of scope for now.